src/main/java/ReadWriteFile.java

```diff
@@ -0,0 +1,73 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadWriteFile {
+
+    public static void readTaskList(ArrayList<Task> taskList) throws DukeException {
+        java.nio.file.Path path = java.nio.file.Paths.get("data").resolve("duke.txt");
+        boolean dirExists = java.nio.file.Files.exists(path);
```

Comment:
According to the Java coding standard guide, under the Naming section, it says that boolean variables/method should sound like booleans. Should the variable name be maybe renamed to `boolean doesDirExists` or `boolean isDirExists` instead of `boolean dirExists`?

Reply:
You're right thanks!